modules/standard/CommDiagnostics.chpl

```diff
@@ -211,53 +211,53 @@ module CommDiagnostics
     /*
       blocking GETs, in which initiator waits for completion
      */
-    var get: uint(64);
+    var get: int(64);
```

Comment:
The runtime definition of these will have to change too, right? At the very least I think you'll have to change the `_chpl_commDiagnostics` struct declaration, and change `chpl_comm_diags_copy` to cast the result of the atomic read to an int. Might also be nice to just convert the underlying atomic counters to ints too, but I don't think that's required. I think, but am not positive, that this code is all encapsulated in runtime/include/chpl-comm-diags.h and we don't leak types into the comm implementations.

Reply:
Oh good point, I always forget that this is a manually reflected data structure (grumble).  Probably not surprising, but the code seemed to "just work" without additional changes which probably makes sense since we're just copying bits and only using the bottom 63 bits in practice?  (not saying I won't change the runtime, just that it had passed testing).